nilearn/tests/test_extmath.py

```diff
@@ -8,11 +8,16 @@
 
 from nilearn._utils.extmath import fast_abs_percentile, is_spd
 
+from sklearn.datasets import load_iris
+from ..image import mean_img
+from ..decoding.tests.test_same_api import to_niimgs
+from ..plotting.find_cuts import find_xyz_cut_coords
+
 
 def test_fast_abs_percentile():
     data = np.arange(1, 100)
```

Comment:
On Wed, Dec 2, 2015 at 5:16 PM, Loïc Estève <notifications@github.com>
wrote:

> In nilearn/tests/test_extmath.py
> <https://github.com/nilearn/nilearn/pull/875#discussion_r46435121>:
>
> > @@ -8,11 +8,16 @@
> >
> >  from nilearn._utils.extmath import fast_abs_percentile, is_spd
> >
> > +from sklearn.datasets import load_iris
> > +from ..image import mean_img
> > +from ..decoding.tests.test_same_api import to_niimgs
> > +from ..plotting.find_cuts import find_xyz_cut_coords
> > +
> >
> >  def test_fast_abs_percentile():
> >      data = np.arange(1, 100)
>
> Can you shuffle it? This is the reason the test were passing before with
> the completely broken implementation of fast_abs_percentile.
>
Good catch. Worst still the data array is not even used...

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nilearn/nilearn/pull/875/files#r46435121>.
>



-- 
Elvis Dohmatob,
*sent from my  potato...*
http://dohmatob.blogspot.fr/, https://team.inria.fr/parietal/elvis/,
https://github.com/dohmatob